core/jobs/src/main/java/com/cognifide/aet/job/common/collectors/screen/ScreenCollector.java

```diff
@@ -56,12 +62,27 @@
 
   private final CollectorProperties properties;
 
+  private String namesOfExcludeElements;
```

Comment:
Consider changing the name of this field - it hold a CSS selector of elements to exclude, so maybe something like `excludeCssSelector`?